GenderPayGap.WebUI/Controllers/Admin/AdminDatabaseIntegrityChecksController.cs

```diff
@@ -261,11 +258,8 @@ public async Task<IActionResult> PrivateSectorOrganisationsWithAPublicSectorType
 
             foreach (Organisation organisation in activePrivateOrganisations)
             {
-                bool isInScope =
-                    organisation.LatestScope.Status == ScopeRowStatuses.Active
-                    && (organisation.LatestScope.ScopeStatus == ScopeStatuses.InScope
-                        || organisation.LatestScope.ScopeStatus == ScopeStatuses.PresumedInScope);
-
+                bool isInScope = organisation.GetCurrentScope().ScopeStatus == ScopeStatuses.InScope ||
```

Comment:
What happens here if the current scope is not active? I assume it returns false, but just want to double check

Reply:
Good spot - it might throw an error, so I've added a null check